ports/kf5crash/portfile.cmake

```
@@ -0,0 +1,29 @@
+vcpkg_from_github(
```

```suggestion
vcpkg_fail_port_install(ON_TARGET "osx" "uwp" "windows")

vcpkg_from_github(
```
I noticed that you add `Supports: linux ` in CONTROL file for this port. It would be better to add the message here.